wgpu-hal/src/gles/queue.rs

```diff
@@ -333,7 +333,18 @@ impl super::Queue {
                 }
 
                 gl.bind_texture(dst_target, Some(dst));
-                if is_layered_target(dst_target) {
+                if dst_array_layer_count == 6 {
```

Comment:
Do we have other such assumptions in the GLES backend? I'm just worrieed that this is a pretty big footgun.

Reply:
We have: 

https://github.com/gfx-rs/wgpu/blob/46420212d71edb6daa1337284f6d94efa7c4ceb3/wgpu-hal/src/gles/device.rs#L568

we could copy an `is_cubemap` bool into the `Texture` and then into this command.

Reply:
Yeah that would probably be the most consistent

Reply:
Done